spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/JdbcChannelMessageStore.java

```diff
@@ -392,6 +394,9 @@ public void afterPropertiesSet() throws Exception {
 			logger.warn("The jdbcTemplate's fetchsize is not 1. This may cause FIFO issues with Oracle databases.");
 		}
 
+		if (this.beanFactory != null) {
+			this.messageBuilderFactory = IntegrationUtils.getMessageBuilderFactory(this.beanFactory);
```

Comment:
Actually, I am not sure these need to be lazy at all; we're fetching 1 bean which has no other deps.

Shouldn't we just remove the usages in setBeanFactory() and move to `afterPropertiesSet()` instead of lazy loading?